[Doc improvement] Document recommended length of labels (#11260)
MicrosoftDocs/msteams-docs: msteams-platform/resources/schema/manifest-schema-dev-preview.md

|`requirements.formFactors`| Array of enums | | | Identifies the form factors that support the add-in. <br>Supported values: `mobile`, `desktop`|
|`tabs`| Array | |✔️| Configures the custom tabs on the Microsoft 365 application ribbon. |
|`tabs.id`| String | 64 characters | | Specifies the ID for the tab within the app.|
|`tabs.builtinTabId`| String | 64 characters | | Specifies the ID of a built-in Office ribbon tab. The possible values vary by Office host application. Outlook add-ins are only supported and the allowed value for Outlook is "TabDefault". The default tab depends on where the Outlook add-in is surfaced, as determined in the "extensions.ribbons.contexts" property. In the main Outlook window, it's the **Home** tab, in a message window, it's the **Message** tab, and in a meeting window, it's the **Meeting** tab. |

> |`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the tab fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|

Reviewer: ```suggestion
|`tabs.label`| String | 64 characters | | Specifies the text displayed for the tab. Despite the maximum length being 64 characters, for the tab to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

|`tabs.position`| Object | | | Configures the position of the custom tab relative to other tabs on the ribbon.|
|`tabs.position.builtinTabId`| String | 64 characters | ✔️ | Specifies the ID of the built-in tab that the custom tab should be positioned next to. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|
|`tabs.position.align`| String enum | | ✔️ | Defines the alignment of custom tab relative to the specified built-in tab. <br>Supported values: `after`, `before`|
|`tabs.groups`| Array | | | Defines groups of controls on a ribbon tab on a non-mobile device. For mobile devices, see `tabs.customMobileRibbonGroups` below.|
|`tabs.groups.id`| String |64 characters | | Specifies the ID for the tab group within the app. It must be different from any built-in group ID in the Microsoft 365 application and any other custom group.|

> |`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the group of controls fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|

Reviewer: ```suggestion
|`tabs.groups.label`| String | 64 characters | | Specifies the text displayed for the group. Despite the maximum length being 64 characters, for the group of controls to align correctly in the ribbon, we recommend you limit the label to 16 characters.|
```

|`tabs.groups.controls.items.overriddenByRibbonApi`| Boolean | | | Specifies whether a group, button, menu, or menu item hidden on application and platform combinations, which support the API ([Office.ribbon.requestCreateControls](/javascript/api/office/office.ribbon#office-office-ribbon-requestcreatecontrols-member(1))) that installs custom contextual tabs on the ribbon. <br>Default value: `false`|
|`tabs.groups.controls.type`| String | | ✔️ | Defines the control type. <br>Supported values: `button`, `menu`|
|`tabs.groups.controls.builtinControlId`| String | 64 characters | ✔️ | Specifies the ID of an existing Microsoft 365 control. For more information, see [find the IDs of controls and control groups](/office/dev/add-ins/design/built-in-button-integration#find-the-ids-of-controls-and-control-groups).|

> |`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control.<br>**Recommendation**: Although the maximum length is 64 characters, to ensure that the control fits correctly in the ribbon, it's a good practice to limit the label to 16 characters.|

Reviewer: ```suggestion
|`tabs.groups.controls.label`| String | 64 characters | ✔️ | Specifies the text displayed for the control. Despite the maximum length being 64 characters, for the control to align correctly in the ribbon, we recommend you limit the label to 16 characters. |
```
